docs/how-to-write-contract.md

```diff
@@ -73,6 +72,11 @@ A contract might face some `RuntimeException` when interacting with `Ledger`, bu
 This contract will just create or update the state of the specified asset, so it doesn't need to return anything to the requester. So in this case, it can return `null`.
 If you want to return something to a requester, you can return an arbitrary `JsonObject`.
 
+### Exception handling
+
+Note that you should not do any exception handling in contracts except for throwing `ContractContextException` as mentioned above.
+Thus, `Ledger` might throw some runtime (unchecked) exceptions in case it can not proceed for some reasons, but the exceptions should not be caught. Exceptions are handled properly by the outside of contracts and will be used to identify if it is retryable or not in the later version.
```

Comment:
```suggestion
Thus, `Ledger` might throw some runtime (unchecked) exceptions in case it can not proceed for some reason, but the exceptions should not be caught. Exceptions are handled properly by the outside of contracts and will be used to identify if it is retryable or not in the later version.
```

Also, I am not exactly sure what you are trying to say in the second sentence. Exceptions should be handled outside of the contracts in later versions? Not now?

Reply:
Exceptions are properly handled now, but not used to identify if it needs to retry.
Once `Ordering Service` is introduced, consumers will need to retry based on the exception.

Probably the current sentence is misleading. How about this simply ?
```
Exceptions are handled properly by the outside of contracts.
```

Reply:
That looks good. One small change: "Exceptions are handled properly outside of contracts."

Reply:
Thank you ! English is difficult :(

Reply:
Updated.